extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/spark/kyuubi/lineage/LineageConf.scala

```diff
@@ -46,6 +46,14 @@ object LineageConf {
       "Unsupported lineage dispatchers")
     .createWithDefault(Seq(LineageDispatcherType.SPARK_EVENT.toString))
 
+  val COLLECT_INPUT_TABLES_BY_PLAN_ENABLED =
+    ConfigBuilder("spark.kyuubi.plugin.lineage.collectInputTablesByPlan.enabled")
+      .doc("Whether to collect input tables by the plan." +
+        " By Default collect input tables by the column lineages.")
+      .version("1.11.0")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
this is more like a bug fix, let's enable it by default and mark it `internal`

Reply:
Thank you for review, I modified the default value and config name, see if it is appropriate